src/mocks/handlers/index.ts

```diff
@@ -8,9 +8,17 @@ import * as gallery from './gallery';
 import * as credits from './credits';
 import * as album from './album';
 
+// import axios from 'axios';
+// import MockAdapter from 'axios-mock-adapter';
+// const mock = new MockAdapter(axios);
```

Comment:
🤔

Reply:
Sorry, I didn't realize that these lines were in this file. This should also be removed.